DatadogRUM/Sources/RUMVitals/RenderLoop/ViewHitchesReader.swift

```diff
@@ -0,0 +1,94 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import UIKit
+
+public typealias Hitch = (start: Int64, duration: Int64)
+
+public protocol ViewHitchesMetric {
+    var hitchesDuration: Double { get }
+    var hitches: [Hitch] { get }
+}
```

Comment:
blocking/ It doesn't seem to be needed in RUM's public API, no?
```suggestion
internal typealias Hitch = (start: Int64, duration: Int64)

internal protocol ViewHitchesMetric {
    var hitchesDuration: Double { get }
    var hitches: [Hitch] { get }
}
```

Reply:
good catch. ty